File: app/src/main/java/io/neoterm/frontend/completion/OnAutoCompleteListener.java
package io.neoterm.frontend.completion;

/**
 * @author Kiva
 * @version 1.0
 */
public interface OnAutoCompleteListener {
    void onAutoComplete(String newText);

    void onKeyCode(int keyCode, int keyMod);

    void onCleanUp();
}

File: app/src/main/java/io/neoterm/frontend/completion/listener/OnAutoCompleteListener.kt
package io.neoterm.frontend.completion.listener

/**
 * @author Kiva
 * *
 * @version 1.0
 */
interface OnAutoCompleteListener {
  fun onCompletionRequired(newText: String?)

  fun onKeyCode(keyCode: Int, keyMod: Int)

  fun onCleanUp()

  fun onFinishCompletion(): Boolean
}
